src/browser/cdp/selectivity/testplane-selectivity.ts

```diff
@@ -2,10 +2,8 @@ import path from "path";
 import { Module as UntypedModule } from "module";
 import { AsyncLocalStorage } from "async_hooks";
 
-const TypedModule = UntypedModule as unknown as {
-    _load: (...args: any) => unknown; // eslint-disable-line @typescript-eslint/no-explicit-any
-    _resolveFilename: (...args: any) => string; // eslint-disable-line @typescript-eslint/no-explicit-any
-};
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const TypedModule = UntypedModule as unknown as { _resolveFilename: (...args: any) => string | void };
```

Comment:
We dont care about arguments, but return type is "string | void", according to node source:
https://github.com/nodejs/node/blob/d4a282b30231d32d36e71c1f28d90e56ee4b9ea8/lib/internal/modules/cjs/loader.js#L1334-L1336